internal/proxy/look_aside_balancer.go

```diff
@@ -37,6 +37,7 @@ import (
 
 var (
 	checkQueryNodeHealthInterval = 500 * time.Millisecond
+	CostMetricsExpireTime        = 1000 * time.Millisecond
```

Comment:
Do we need it to be configurable?

Reply:
the same as checkQueryNodeHealthInterval

Reply:
the internal policy, config it's value is no meaning for users.

Reply:
It is not really useful for end users, mainly during the operation and maintenance phase. Will adjusting parameters be helpful when problems occur? 🤔

Reply:
`CostMetricsExpireTime` doesn't take effect during problems occur. it just worked in multi-replica cases with very low search/query workload.

`checkQueryNodeHealthInterval` will decide the latency that proxy knows query node is unavailable. so to reduce the latency, we set a nearly smallest and reasonable value here. give it a larger value doesn't make sense